real_time/arduino_src/radio_buggy_mega/Makefile

```diff
@@ -328,7 +330,7 @@ ALL_ASFLAGS = -mmcu=$(MCU) -I. -x assembler-with-cpp $(ASFLAGS)
 
 
 # Default target.
-all: begin gccversion sizebefore build sizeafter end
+all: begin gccversion sizebefore fingerprint build sizeafter end
 

```

Comment:
Line 348 and 352, respectively. They don't do anything other than output a message to the terminal